### PR TITLE
saner default cursor

### PR DESCRIPTION
**Pull request by lieuxnoir**

### CODE REVIEW

**Comment by dylanaraps** [comment]
Screenshot of the difference?

**Comment by lieuxnoir** [comment]
http://0x0.st/zx2O.png

File: sowm.c [hunk 1]
```diff
@@ -341,6 +341,7 @@ int main(void) {
     sh   = XDisplayHeight(d, s);
 
     key_grab();
+    XDefineCursor(d, root, XCreateFontCursor(d, 68));
```
Comment (dylanaraps) [inline sowm.c]: Move it here:

```patch
diff --git a/sowm.c b/sowm.c
index b8bedc6..63c2eaa 100644
--- a/sowm.c
+++ b/sowm.c
@@ -358,6 +358,8 @@ int main(void) {
         ButtonPressMask|ButtonReleaseMask|PointerMotionMask,
         GrabModeAsync, GrabModeAsync, None, None);

+    XDefineCursor(d, root, XCreateFontCursor(d, 68));
+
     while(1 && !XNextEvent(d, &ev))
         if (events[ev.type]) events[ev.type](&ev);
 }
```

Comment (lieuxnoir) [reply sowm.c]: done :D

**Comment by dylanaraps** [comment]
LGTM :+1:

Thanks!

**Pull request merged**